packages/backend/src/models/OrganizationAllowedEmailDomainsModel.ts

```diff
@@ -23,7 +27,11 @@ export class OrganizationAllowedEmailDomainsModel {
             organizationUuid:
                 dbOrganizationAllowedEmailDomains.organization_uuid,
             emailDomains: dbOrganizationAllowedEmailDomains.email_domains,
-            role: dbOrganizationAllowedEmailDomains.role,
+            role: isAllowedEmailDomainsRole(
```

Comment:
Do we need this check? I assumed the controller should do the validation for us

Reply:
The column allows any org type. I could have added a migration to create a new table and change this column reference but felt like overkill.